docs/supported-browsers.md

```diff
@@ -0,0 +1,30 @@
+# Supported Browsers
+
+### **Date of last revision:** 01-02-2018 (DD-MM-YYYY)
+### **Date of next revision:** 03-09-2018 (DD-MM-YYYY)
+
+## The list of supported browsers is:
+
+* Chrome 38 and above
+* Safari 8 and above
+* IE 11
+* Microsoft Edge all versions
+* Firefox 44 and above
+* Samsung 4 and above
+* Opera 25 and above
+
+Note about Android WebView: If someone reports a issue we are going to fix it but we are not going proactively test it.
```

Comment:
Maybe we could add something like:

"iOS Webviews should be mostly equivalent to Safari, and Android Webviews (for recent versions of Android) should be mostly equivalent to Chrome. Based on this premise we have concluded that by supporting their parent browsers we implicitly include support for these webviews as well. However, if we discover that there are problems with this approach then we will revise it."

Reply:
I like Jamie's suggestion